clang/test/SemaCXX/cxx2a-consteval.cpp

```diff
@@ -1321,4 +1321,39 @@ namespace GH139160{
                                                 // expected-note@-2  {{non-constexpr function 'make_struct' cannot be used in a constant expression}}
 };
 
+// taken from: https://github.com/llvm/llvm-project/issues/139818
+namespace GH139818{
```

Comment:
GH139818 already reflects the source.

```suggestion
namespace GH139818 {
```